pom.xml

```diff
@@ -521,37 +521,87 @@
+                                <licenseMerge>
+                                    BSD-3-Clause |
+                                    BSD 3-Clause License |
+                                    BSD |
+                                    BSD licence |
+                                    The BSD License |
+                                    3-Clause BSD License |
+                                    New BSD license |
+                                    The New BSD License |
+                                    Revised BSD |
+                                    BSD New license |
+                                    BSD 3-clause
+                                </licenseMerge>
+                                <licenseMerge>
+                                    CC0-1.0 |
+                                    CC0 |
+                                    Public Domain, per Creative Commons CC0
+                                </licenseMerge>
                                 <licenseMerge>
-                                    Apache License, Version 2.0|Apache License v2.0|
-                                    Apache Software License 2.0|The Apache Software License, Version 2.0|
-                                    Apache License, version 2.0|The Apache Software License, version 2.0|
-                                    Apache 2|Apache 2.0|Apache-2.0|Apache License 2.0|The Apache License, Version 2.0|
-                                    Apache Software License - Version 2.0|Apache License Version 2.0
+                                    CDDL |
+                                    CDDL + GPLv2 with classpath exception |
+                                    CDDL License |
+                                    CDDL/GPLv2+CE
                                 </licenseMerge>
-                                <licenseMerge>MIT License|MIT|The MIT License|MIT license|The MIT License (MIT)</licenseMerge>
                                 <licenseMerge>
-                                    BSD 3-Clause License|BSD License|BSD|BSD licence|The BSD License|3-Clause BSD License|
-                                    New BSD License|New BSD license|The New BSD License|Revised BSD|BSD New license|
-                                    BSD-3-Clause|BSD 3-clause
+                                    Eclipse Distribution License - v 1.0 |
+                                    EDL 1.0
                                 </licenseMerge>
-                                <licenseMerge>BSD 2-Clause License|BSD-2-Clause|BSD 2-Clause license</licenseMerge>
-                                <licenseMerge>CC0|Public Domain, per Creative Commons CC0</licenseMerge>
                                 <licenseMerge>
-                                    CDDL|CDDL + GPLv2 with classpath exception|CDDL License|CDDL/GPLv2+CE
+                                    EPL-2.0 |
+                                    EPL 2.0 |
+                                    Eclipse Public License - Version 1.0
```

Comment:
Wrong version.

Reply:
Good question @kwart 
1. I kept everything from before as I didn't want to rock the boat so to speak (more testing will be needed). I think its possible some are unused tbh but to remove each one I would need to recompile hazelcast (and possibly hazelcast-enterprise as we do update some part of the license in there) for every removal which will take long. How about we keep as is and I raise a separate GH issue to clean-up the list?
2. good catch with the "EPL 2.0 | Eclipse Public License - Version 1.0". I will fix that shortly and push again

Reply:
FYI...changed pushed https://github.com/hazelcast/hazelcast/pull/25150/commits/dddb016b9afd7f192745f1bfbd215bb28ef2368e

Reply:
Pushed another update after further review comments from internal stakeholder

@kwart please approve if you are ok?